.github/workflows/wpt.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will check out wpt and run the WebDriver BiDi tests against our
+# implementation.
+
+name: web-platform-tests
+
+on:
+  pull_request:
+    branches: '*'
+
+jobs:
+  wpt:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - name: Install dependencies
+        run: npm install
+      - name: Build
+        run: npm run build
+      # WPT setup
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Set up virtualenv
+        run: pip install virtualenv
+      - uses: actions/checkout@v2
+        with:
+          repository: web-platform-tests/wpt
+          path: wpt
+          fetch-depth: 1
+      - name: Set up hosts
+        run: ./wpt make-hosts-file | sudo tee -a /etc/hosts
+        working-directory: wpt
+      # TODO: install certs
+      - name: Install Chrome
+        run: ./wpt install chrome browser
+        working-directory: wpt
+      - name: Run tests
+        run: ./wpt run --webdriver-binary=../runBiDiServer.sh --binary _venv3/browsers/nightly/chrome-linux/chrome --log-mach - --log-mach-level info chrome webdriver/tests/bidi/
+        working-directory: wpt
+env:
```

Comment:
Can we add ` cat log.txt` after the tests? It could help investigate the failures.

Reply:
Many servers will be started and they'll clobber each others logs, but yes, I'll add a job to print the log on failure.